File: Python/Power of Three.py
class Solution(object):
    def isPowerOfThree(self, n):
        """
        :type n: int
        :rtype: bool
        """
        if n <= 0:
            return False
        else:
            while n % 3 == 0:
                n /= 3
            if n == 1 or n == 0:
                return True
            return False

File: cpp/Power of Three.cpp
class Solution {
public:
    bool isPowerOfThree(int n) {
        if (n <= 0)
            return false;
        else {
            while (!(n % 3)) {
                n /= 3;
            }
            return n == 1 || n == 0;
        }
    }
};
